README.rst

```diff
@@ -25,7 +27,7 @@ The recommended way to install Silex is through `Composer`_:
 
 .. code-block:: bash
 
-    composer require silex/silex "~2.0"
+    $ composer require silex/silex "~2.0"
```

Comment:
Isn't it possible to reuse the `terminal` code block that is used for the Symfony documentation?